### PR TITLE
D5

### PR DESCRIPTION
**Pull request by Weemizo**
Quick fix, added the link.

### CODE REVIEW

**Comment by Weemizo** [comment]
uQr4UCV6vh

File: src/app/layout.tsx [hunk 1]
```diff
@@ -28,23 +28,23 @@ export default function RootLayout({
...
+              <span className='title'>
+                <Link href="/">
+                  Amber
+                </Link>
+              </span>
               </div>
                 <SearchBar/>
               <div className='nav-right'>
                 <Button>
                 <Icon src='internal/side-bar.svg' size='2rem'/>
```
Comment (Ph0enixKM) [inline src/app/layout.tsx:43]: @Weemizo Can you indent this icon and all other icons in this page please?

```suggestion
                  <Icon src='internal/side-bar.svg' size='2rem'/>
```

**Comment by Ph0enixKM** [comment]
@Weemizo Everything looks great. Just two minor requests regarding the code quality.
1. Please indent the `<Icon/>` components inside of the buttons to match the indentation.
2. Please move the contents of the `<nav/>` to other file and make a new component out of it. Perhaps `Navigation` or something similar.

Good job. I like the overall solutions and fixes that you applied to the code.

File: src/app/layout.tsx (hunk 1 above)
Comment (Weemizo) [reply src/app/layout.tsx:43]: Sure thing

**Comment by Ph0enixKM** [comment]
LGTM, merging.

**Review by Ph0enixKM — APPROVED** [review]
LGTM, merging.

**Pull request merged**